README.md

```diff
@@ -213,33 +214,26 @@ or
 bundle exec cap prod db_seed # for the prod servers
 ```
 
-### Populate the catalog
+### Populate the catalog for a single storage root
 
 In console, start by finding the storage root.
 
 ```ruby
 msr = MoabStorageRoot.find_by!(name: name)
-Audit::MoabToCatalog.seed_catalog_for_dir(msr.storage_location)
-```
-
-Or for all roots:
-```ruby
-MoabStorageRoot.find_each { |msr| Audit::MoabToCatalog.seed_catalog_for_dir(msr.storage_location) }
+CatalogUtils.populate_catalog_for_dir(msr.storage_location)
 ```
 
 ## Deploying
 
 Capistrano is used to deploy.  You will need SSH access to the targeted servers, via `kinit` and VPN.
 
 ```sh
-bundle exec cap stage deploy # for the stage servers
+bundle exec cap ENV deploy # e.g. bundle exec cap qa deploy
 ```
 
-Or:
+## Jobs
 
-```sh
-bundle exec cap prod deploy # for the prod servers
-```
+Sidekiq is run on one or more worker VMs to handle queued jobs.
```

Comment:
the `### Sidekiq` section below felt like it was better placed in a `Jobs` section than in the `Deploying` section.